kernels/portable/cpu/util/reduce_util.h

```diff
@@ -24,9 +26,12 @@ void apply_on_flat_ix_with_stride_and_base(
     const size_t base,
     const size_t start,
     const size_t end) {
-  for (size_t i = start; i <= end; i++) {
```

Comment:
note that this loop should've been migrated to irange already, I just haven't gotten around to that migration yet